packages/webapp/src/components/SensorReadingsLineChart/index.jsx

```diff
@@ -93,12 +93,18 @@ const PureSensorReadingsLineChart = ({
   const renderDateOnXAxis = (tickProps) => {
     const { x, y, payload, index } = tickProps;
     const { value, offset } = payload;
-    const displayDate = moment(value).format('ddd, MMM DD');
+    const displayDate = moment(value).format('DD');
+    const displayDay = moment(value).format('ddd');
```

Comment:
In general, we are trying to move away from using moment in LiteFarm as it is now deprecated (See `deprecate-momentjs` [here](https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1191575576/Pre-integration+peer+review)). Can you refactor this to not use moment?

Reply:
Done